cygnus-common/pom.xml

```diff
@@ -82,7 +82,7 @@
         <dependency>
             <groupId>log4j</groupId>
             <artifactId>log4j</artifactId>
-            <version>1.2.17</version>
+            <version>1.2.27</version>
```

Comment:
Maybe should be documented in the CNR (as for instance "[cygnus-common] Upgrade MongoDB driver from 3.0.4 to 3.11.0")

Reply:
But there is not still version 1.2.27!